bin/check-redis-keys.rb

```diff
@@ -68,6 +68,12 @@ class RedisKeysCheck < Sensu::Plugin::Check::CLI
          required: false,
          default: '*'
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
```

Comment:
This could be improved a bit to say "Returns the following exit status for Redis connection failures" which seems a bit more descriptive and does not me to jump to the code to understand what you mean.